contracts/test/Wallet.sol

```diff
@@ -27,4 +27,12 @@ contract Wallet is Ownable {
         emit Pay(_beneficiary, amount);
     }
 
+    function genericCall(address _contract, bytes memory _encodedABI)
+    public
+    returns(bool success, bytes memory returnValue) {
+       // solhint-disable-next-line avoid-low-level-calls
+        (success, returnValue) = _contract.call(_encodedABI);
```

Comment:
Why instead of sending the _encodeABI we execute the withdraw function with the signature that we already know it will have?
```
(success, returnValue) = _contract.call(abi.encodeWithSignature("withdraw()"));
```

Doing this we make sure that the ONLY function that can be executed is the `withdraw()`

Reply:
It is a public generic function which can call any contract specific Abis.
this is just a test helper contract and is not part of any dao.